src/marketing/utilities/borders.scss

```diff
@@ -1,9 +1,11 @@
 // Border utilities
 
 // XXX If you're looking for responsive border utilities, they've moved to
-// ../../primer-utilities/lib/borders.scss
+// ../../utilities/borders.scss
 
-/* Use with .border to turn the border rgba white 0.15 */
-.border-white-fade {
-  border-color: $white-fade-15 !important;
-}
+/* Use with .border to turn the border white with transparency */
+.border-white-fade-15 { border-color: $white-fade-15 !important; }
+.border-white-fade-30 { border-color: $white-fade-30 !important; }
+.border-white-fade-50 { border-color: $white-fade-50 !important; }
+.border-white-fade-70 { border-color: $white-fade-70 !important; }
+.border-white-fade-85 { border-color: $white-fade-85 !important; }
```

Comment:
I'll need these in `core` so I can use them on the header. Can you move them there?